lib/ansible/modules/source_control/git.py

```diff
@@ -1023,13 +1059,22 @@ def main():
     # call run_command()
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
 
+    if separate_git_dir:
```

Comment:
Is it intentional that you don't match an empty string?

Reply:
empty `separate_git_dir` makes no sense so I ignore that.